studio/tests/remote_worker_test.py

```diff
@@ -46,7 +46,7 @@ def test_remote_worker(self):
             config_name='test_config_http_client.yaml',
             test_script='tf_hello_world.py',
             script_args=['arg0'],
-            expected_output='[ 2.  6.]',
+            expected_output='[ 2.0 6.0 ]',
```

Comment:
remote worker tests - those are skipped in order not too use docker in travis, but these changes are needed when being run manually